src/github.com/getlantern/analytics/analytics.go

```diff
@@ -12,11 +12,9 @@ import (
 )
 
 const (
-	ApiEndpoint          = `https://ssl.google-analytics.com/collect`
-	ProtocolVersion      = "1"
-	DefaultClientVersion = "1"
-	TrackingId           = "UA-21815217-2"
-	DefaultClientId      = "555"
```

Comment:
Furthermore, can we use `instanceId` as `ClientId`? For now, it's empty except our hosted servers.